openwell/questioner
# #163722542 creation of three routes and code refactoring

#### What does this PR do?
- This PR allows the user to get rsvp response
#### Description of Task to be completed?
- When a GET request is sent to `localhost:3000/api/v1/meetups/rsvps`, with a user token
- A successful request should return a status of 200 with the user rsvp to all meetups
#### How should this be manually tested?
- Pull this branch [ft-user-able-get-rsvp-163722542](https://github.com/openwell/questioner/tree/ft-user-able-get-rsvp-163722542) off this repository
- On your command line, run npm i to install all dependencies for this app
- On your command line, run npm database and npm start to initialize the app
- App would run on port 3000
- Make a GET request to  `localhost:3000/api/v1/meetups/rsvps` with a user token
- A successful request should return a status of 200 with the user rsvp to a meetup
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
[#163722542](https://www.pivotaltracker.com/story/show/163722542)
[#163788743](https://www.pivotaltracker.com/story/show/163788743)
[#163788763](https://www.pivotaltracker.com/story/show/163788763)
#### Screenshots (if appropriate)
None
#### Questions:
None
